draft-ietf-core-oscore-groupcomm.md

```diff
@@ -1165,6 +1252,8 @@ For each ongoing observation, the server can help the client to synchronize, by
 
 If there is a known upper limit to the duration of a group rekeying, the server SHOULD include the 'kid context' parameter during that time. Otherwise, the server SHOULD include it until the Max-Age has expired for the last notification sent before the installation of the new Security Context.
 
+As per {{sec-replay-protection-non-notifications}}, the server MUST NOT reply to a group request with 2.xx responses of which some are notifications and some are not. That is, if the server receives an observation request and does not register the observation, then any following 2.xx response from the server to that request MUST NOT be a notification. Also, if the server receives an observation request and registers the observation, then any following 2.xx response from the server to that request MUST be a notification.
```

Comment:
Could be rephrased to avoid double negative?

_That is, if the server receives an observation request and does not register the observation, then any following 2.xx response from the server to that request MUST NOT be a notification._

-> 

_That is, if the server receives an observation request and registers the observation, then any following 2.xx response from the server to that request MUST be a notification._

Reply:
Sounds better, thanks!